manager/apis/app/v1alpha1/fybrikapplication_types.go

```diff
@@ -5,30 +5,27 @@ package v1alpha1
 
 import (
 	"fybrik.io/fybrik/pkg/model/taxonomy"
+	"github.com/c2h5oh/datasize"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// CatalogRequirements contain the specifics for catalogging the data asset
-type CatalogRequirements struct {
-	// CatalogService specifies the datacatalog service that will be used for catalogging the data into.
+// FlowRequirements include the requirements specific to the flow
+// Note: Implicit copies done for data plane optimization by Fybrik do not use these parameters
+type FlowRequirements struct {
+	// Catalog indicates that the data asset must be cataloged, and in which catalog to register it
 	// +optional
-	CatalogService string `json:"service,omitempty"`
+	Catalog string `json:"catalog,omitempty"`
 
-	// CatalogID specifies the catalog where the data will be cataloged.
+	// Storage estimate indicates the estimated amount of storage required when writing new data.
```

Comment:
The comment will be propagated to the Web site, should we add examples of the estimated storage,  MB, GB, TB ...

Reply:
Catalog is used only for registration of a new asset. There is no need for SourceCatalogID as datasetID includes this information.

Reply:
Adding comment about storage units